paper/src/main/java/com/denizenscript/denizen/paper/events/TargetBlockHitScriptEvent.java

```diff
@@ -0,0 +1,83 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import io.papermc.paper.event.block.TargetHitEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+
+public class TargetBlockHitScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // targetblock hit
+    //
+    // @Location true
+    //
+    // @Group Paper
+    //
+    // @Plugin Paper
+    //
+    // @Triggers when a target block is hit by a projectile such as an arrow.
+    //
+    // @Context
+    // <context.projectile> returns an EntityTag of the projectile.
+    // <context.shooter> returns a EntityTag of the shooter of the projectile.
+    // <context.hit_block> returns a LocationTag of the block.
+    // <context.hit_face> returns a LocationTag of the face.
```

Comment:
These were copied from the `projectile hits` event, but cut short -- should probably just restore the full text from the other events docs into here